standalone-metastore/metastore-rest-catalog/pom.xml

```diff
@@ -224,9 +230,25 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-exec</artifactId>
```

Comment:
same, we shouldn't remove all hive dependencies from the standalone-metastore modules